backend/tests/assetlinks.test.ts

```diff
@@ -0,0 +1,49 @@
+import { afterAll, beforeAll, describe, expect, it } from '@jest/globals';
+import request from 'supertest';
+import { Server } from 'http';
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+process.env.VIDEO_SERVICE_PROVIDER = 'opentok';
+
+const startServer = (await import('../server')).default;
+const fileName = fileURLToPath(import.meta.url);
+const dirName = path.dirname(fileName);
+
+describe('GET /.well-known/assetlinks.json', () => {
+  let server: Server;
+
+  beforeAll(async () => {
+    server = await startServer();
+  });
+
+  afterAll((done) => {
+    server.close(done);
+  });
+
+  it('returns a 200 status code', async () => {
+    const res = await request(server).get('/.well-known/assetlinks.json');
+    expect(res.statusCode).toEqual(200);
+  });
+
+  it('returns the correct Content-Type header', async () => {
+    const res = await request(server).get('/.well-known/assetlinks.json');
+    expect(res.headers['content-type']).toMatch(/application\/json/);
```

Comment:
Nit: a regular string comparison (i.e., of "application/json") would be simpler.

Friends don't let friends regex, unless they need to :-)